frontend/src/components/RideModal/ridemodal.module.css

```diff
@@ -4,6 +4,11 @@
   align-items: center;
 }
 
+.error {
+  color: #FF647C;
```

Comment:
Make sure you use #EB0023 for error colors instead of this, the #FF647C is too light against a white background